cpp/benchmarks/io/json/json_reader_input.cpp

```diff
@@ -26,15 +26,17 @@
 
 // Size of the data in the benchmark dataframe; chosen to be low enough to allow benchmarks to
 // run on most GPUs, but large enough to allow highest throughput
-constexpr size_t data_size         = 512 << 20;
```

Comment:
Let’s keep this constexpr variable here (maybe rename it) and use it as the default value for `datasize` below

Reply:
Good idea - fixed!